opm/output/eclipse/Summary.cpp

```diff
@@ -384,8 +402,16 @@ template<rt phase , bool injection>
 quantity region_rate( const fn_args& args ) {
     double sum = 0;
     const auto& well_completions = args.regionCache.completions( args.num );
+
     for (const auto& pair : well_completions) {
-        double rate = args.wells.get( pair.first , pair.second , phase );
+        auto it = std::find_if( args.eff_factors.begin(), args.eff_factors.end(),
```

Comment:
This code block is repeated three times; I guess it could be a free function:
```C++
double get_efac( std::vector<std::pair<std::string,double>>, const std::string& elem)  {
...
}
```